src/pages/ClusterPage/ClusterPage.js

```diff
@@ -27,8 +23,21 @@ import {
 	CLUSTER_PLANS,
 	ARC_PLANS,
 } from './utils';
+import { mediaKey } from '../../utils/media';
+import DeleteClusterModal from './components/DeleteClusterModal';
+import StripeCheckout from '../../components/StripeCheckout';
+import { ansibleMachineMarks } from './new';
+import { machineMarks as arcMachineMarks } from './NewMyCluster';
+import { clusterContainer, clustersList, bannerContainer } from './styles';
+
 import { regions } from './utils/regions';
 
+// overiding the fromNow() method in moment to return the diff only in hours
+moment.fn.fromNow = function fromNow() {
```

Comment:
Instead of overriding why can't we use `moment().diff(this, 'hours');` directly?

This could create a side effect when `fromNow` is used in another component.

You can create, getHoursDiff helper if required.